bin/next-start

```diff
@@ -10,11 +10,13 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'production'
 const argv = parseArgs(process.argv.slice(2), {
   alias: {
     h: 'help',
+    H: 'hostname',
     p: 'port'
   },
   boolean: ['h'],
   default: {
-    p: 3000
+    p: 3000,
+    hostname: 'localhost'
```

Comment:
@jessehattabaugh great though actually. Only now I'm remembering that when you pass `-H`  without a hostname it will actually turn into `true`